extensions/http/src/jolie/net/HttpProtocol.java

```diff
@@ -191,6 +191,7 @@
 		private static final String COMPRESSION_TYPES = "compressionTypes";
 		private static final String REQUEST_COMPRESSION = "requestCompression";
 		private static final String FORMAT = "format";
+                private static final String RESPONSE_HEADER = "response_header";
```

Comment:
Indentation seems wrong.

Also, response_header should be plural (since we have more than one header) and it looks like we're using camelCase in the rest of the parameters, so that gives us "responseHeaders". Can you switch?